virttest/utils_misc.py

```diff
@@ -4705,3 +4698,81 @@ def _md5(fd):
         if _md5(fd_a) == _md5(fd_b):
             return True
     return False
+
+
+def verify_sev_host(params):
+    """
+    Verify if specific sev feature is enabled on the host
+
+    :param params: Dictionary with the test parameters
+    """
+    cvm_type = params["vm_secure_guest_type"]
+    cvm_module_path = params["cvm_module_path"]
+    LOG.info(f"Verifying cvm {cvm_type} capability enablement on host")
+    try:
+        if os.path.exists(cvm_module_path):
+            with open(cvm_module_path) as f:
+                output = f.read().strip()
+                if output not in params.objects("module_status"):
```

Comment:
Some new parameters are introduced, but without a clear definition(e.g.: `module_status`), I would suggest using them in the function call instead of giving the raw params, to make this be a pure and clear function.

params --> module_name, module_status

BTW, if you only want to ensure the module with correct parameter loaded before test, maybe `kvm_probe_module_parameters` can help you, it will check `kvm-amd` for AMD, `kvm-intel` for Intel, and you can set it in cfg file like `kvm_probe_module_parameters = "sev=1"`